schedule/yam/agama_unattended_keyboard_layout.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: agama_unattended_keyboard_layout
```

Comment:
agama_keyboard_layout_unattended

Reply:
Thanks for review, I renamed the file name.

Reply:
btw, I also renamed the new file  `sle_default_keyboard_layout.jsonnet`  to `sles_default_keyboard_layout.jsonnet` .